files/default/sql/ddl/2.6.0__initial_tables.sql

```diff
@@ -2270,3 +2270,65 @@ CREATE TABLE IF NOT EXISTS `git_repository_remotes` (
 ) ENGINE=ndbcluster AUTO_INCREMENT=6164 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `expectation_suite` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_group_id` INT(11) NOT NULL,
+    `name` VARCHAR(63) NOT NULL,
+    `meta` VARCHAR(1000) DEFAULT "{}",
+    `data_asset_type` VARCHAR(50),
+    `ge_cloud_id` VARCHAR(200),
+    `run_validation` BOOLEAN DEFAULT TRUE,
+    `validation_ingestion_policy` VARCHAR(25),
+    `created_at` timestamp DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `feature_group_suite_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `expectation` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `expectation_suite_id` INT(11) NOT NULL,
+    `expectation_type` VARCHAR(150) NOT NULL,
+    `kwargs` VARCHAR(1000) NOT NULL,
+    `meta` VARCHAR(1000) DEFAULT "{}",
+    PRIMARY KEY (`id`),
+    CONSTRAINT `suite_fk` FOREIGN KEY (`expectation_suite_id`) REFERENCES `expectation_suite` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `great_expectation` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `kwargs_template` VARCHAR(1000) NOT NULL,
+    `expectation_type` VARCHAR(150) NOT NULL,
+    UNIQUE KEY `unique_great_expectation` (`expectation_type`),
+    PRIMARY KEY (`id`)
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `validation_report` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_group_id` INT(11) NOT NULL,
+    `success` BOOLEAN NOT NULL,
+    `validation_time` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+    `evaluation_parameters` VARCHAR(1000) DEFAULT "{}",
+    `meta` VARCHAR(1000) NULL DEFAULT "{}",
+    `statistics` VARCHAR(1000) NOT NULL,
+    `inode_pid` BIGINT(20) NOT NULL,
+    `inode_name` VARCHAR(255) COLLATE latin1_general_cs NOT NULL,
+    `partition_id` BIGINT(20) NOT NULL,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `feature_group_report_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
+    CONSTRAINT `inode_result_fk` FOREIGN KEY (`inode_pid`,`inode_name`,`partition_id`) REFERENCES `hops`.`hdfs_inodes` (`parent_id`,`name`,`partition_id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `validation_result` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `validation_report_id` INT(11) NOT NULL,
+    `expectation_id` INT(11) NOT NULL,
+    `success` BOOLEAN NOT NULL,
+    `result` VARCHAR(1000) NOT NULL,
+    `meta` VARCHAR(1000) DEFAULT "{}",
+    `expectation_config` VARCHAR(2150) NOT NULL,
+    `exception_info` VARCHAR(1000) DEFAULT "{}",
```

Comment:
Or maybe this will be longer than 1k chars?
Wouldn't hurt to raise it to 5 or 10k?
Remember, total max size per row is ~30k.

Reply:
it doesn't contain the full stack trace, just some summary. so it should be enough. you can always download the full report otherwise.